docs/api-reference.md

```diff
@@ -32,21 +32,21 @@ Creates a new Roact fragment with the provided table of elements. Fragments allo
 
 ### Roact.mount
 ```
-Roact.mount(element, [parent, [key]]) -> RoactTree
+Roact.mount(element, [parent, [key]]) -> VirtualTree
```

Comment:
I can switch it back to RoactTree, it's just that the Type we are going to expose in #230 is called `VirtualTree`. So maybe it could be renamed in that other PR?

Reply:
That's a good question! I didn't realize we were exposing it in that PR. We should probably discuss this with @LPGhatguy.

My thoughts are still that `VirtualTree` probably means less to users, but maybe there's another better alternative too.

Reply:
Yes I agree that the term VirtualTree might be questionable to users, we should probably solves this in #230 if it's about to be merged first.